collect_emoji_svg.py

```diff
@@ -75,66 +82,69 @@ def copy_with_rename(src_dir, dst_dir, accept_pred=None, rename=None, verbosity=
     src = os.path.join(src_dir, src_filename)
     dst = os.path.join(dst_dir, dst_filename)
     if os.path.exists(dst):
-      if verbosity > 1:
-        print "Replacing existing file " + dst
+      logging.debug('Replacing existing file %s', dst)
       os.unlink(dst)
       replace_count += 1
     shutil.copy2(src, dst)
-    if verbosity > 1:
-      print "cp -p %s %s" % (src, dst)
+    logging.debug('cp -p %s %s', src, dst)
     count += 1
-  if verbosity:
-    print "Copied/renamed %d files from %s to %s" % (count, src_dir, dst_dir)
-  return count, replace_count
-
+  if logging.getLogger().getEffectiveLevel() >= 20:
```

Comment:
20 feels like kind of a magic number here, could we instead do `>= logging.INFO`?